docs/user-guide/cli-installplugins.md

```diff
@@ -15,38 +15,40 @@ You can install the following Zowe plug-ins:
 
 Use either of the following methods to install plug-ins:
 
-- Install from an online NPM registry. Use this method when your computer ***can*** access the Internet.
+- Install from an online NPM registry. Use this method when your computer can access the Internet.
 
     For more information, see [Installing plug-ins from an online registry](#installing-plug-ins-from-an-online-registry).
 
-- Install from a local package. With this method, you download and install the plug-ins from a bundled set of `.tgz` files. Use this method when your computer ***cannot*** access the Internet.
+- Install from a local package. With this method, you download and install the plug-ins from a bundled set of `.tgz` files. Use this method when your computer cannot access the Internet.
 
     For more information, see [Installing plug-ins from a local package](#installing-plug-ins-from-a-local-package).
 
 ## Installing plug-ins from an online registry
 
-Install Zowe CLI plug-ins on Windows, Mac, and Linux. The procedures in this article assume that you previously installed the core CLI.
+The following procedure assumes that you previously installed Zowe CLI core.
 
-**Follow these steps:**
+To install Zowe CLI plug-ins on Windows, Mac, and Linux:
 
-1. Meet the [software requirements for each plug-in](cli-swreqplugins.md) that you install.
+1. Meet the [software requirements for each plug-in](cli-swreqplugins.md) to be installed.
 
-2.  Issue the following command to install a plug-in from public npm:
+2. Issue the following command to install a plug-in from public npm:
```

Comment:
Be consistent with the earlier procedures. Do not include "Issue the following command to." Change to "Install a plug-in from public npm:"